www/src/components/navigation/githubIcon.astro

```diff
@@ -47,7 +47,11 @@ const isRtl = getIsRtlFromUrl(pathname);
         d="M316.9 18C311.6 7 300.4 0 288.1 0s-23.4 7-28.8 18L195 150.3 51.4 171.5c-12 1.8-22 10.2-25.7 21.7s-.7 24.2 7.9 32.7L137.8 329 113.2 474.7c-2 12 3 24.2 12.9 31.3s23 8 33.8 2.3l128.3-68.5 128.3 68.5c10.8 5.7 23.9 4.9 33.8-2.3s14.9-19.3 12.9-31.3L438.5 329 542.7 225.9c8.6-8.5 11.7-21.2 7.9-32.7s-13.7-19.9-25.7-21.7L381.2 150.3 316.9 18z"
       ></path>
     </svg>
-    {Intl.NumberFormat().format(githubStars)}
+    {
+      Intl.NumberFormat(getLanguageFromURL(pathname), {
+        notation: "compact",
+      }).format(githubStars)
+    }
```

Comment:
I think it's better to do this in the javascript part at the top rather than in the markdown?

Reply:
I would agree with this, but to keep the md code clean. In terms of outcome, it would be the same right?

Reply:
Something like: 
```js
const formattedStars = new Intl.NumberFormat(getLanguageFromURL(pathname), {
  notation: "compact",
}).format(githubStars);
```
and just call formattedStars on the markdown

Reply:
Judging by other references on the repo, seems like the convention is to keep js/md separate as much a possible.

I pushed the changes 👍